openapi/components/schemas/Product.yaml

```
@@ -2,19 +2,14 @@ allOf:
   - $ref: Common/CommonProduct.yaml
   - properties:
       taxCategoryId:
-        description: The product's tax category identifier string.
+        description: |-
+          Tax category of the product.
+          See a complete list of supported tax categories in [TaxJar sales tax API reference](https://developers.taxjar.com/api/reference/#get-list-tax-categories).
```

nitpick: [clause order](https://product.rebilly.dev/writing-style-guide/grammar-and-mechanics/clause-order/)

```suggestion
          For a complete list of supported tax categories, see [TaxJar sales tax API reference](https://developers.taxjar.com/api/reference/#get-list-tax-categories).
          If none of the tax categories from the list are applicable for your product, use the general tax category `00000`.
```